### ISSUE TITLE
AttributeError: 'function' object has no attribute 'tostring'

### ISSUE THREAD

**Issue by mobbSF** [pydicom/deid#52]
Hi, 
Thanks for the opportunity to use this cool library 🙂 

So, everything is going great until the `.save_dicom()` step.  

```
client = DicomCleaner(output_folder='/output', deid=my_deid)
client.clean()
```
scrubbing happens: 
_Scrubbing /Users/me/dicoms/dicom.dcm._
Then,
`client.save_dicom()`
gives
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-77-35491d2123a0> in <module>()
----> 1 client.save_dicom()

/Users/me/anaconda3/envs/python_p27/lib/python2.7/site-packages/deid/dicom/pixels/clean.pyc in save_dicom(self, output_folder, image_type)
    179             dicom_name = self._get_clean_name(output_folder)
    180             dicom = read_file(self.dicom_file,force=True)
--> 181             dicom.PixelData = self.clean.tostring()
    182             dicom.save_as(dicom_name)
    183             return dicom_name

AttributeError: 'function' object has no attribute 'tostring'
```

It looks like the problem is with the `read_file` from pydicom...but I'm assuming the dicoms are readable since they were opened and read in the scrubbing step.  

To experiment, I tried to save as a png:  
`client.save_png()`
and got 
```
Error in callback <function post_execute at 0x111179668> (for post_execute):
UsageError: Invalid GUI request 'pdf', valid ones are:[None, 'osx', 'widget', 'qt5', 'qt', 'nbagg', 'gtk', 'qt4', 'gtk3', 'notebook', 'tk', 'ipympl', 'inline', 'asyncio', 'wx']
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/Users/me/anaconda3/envs/python_p27/lib/python2.7/site-packages/matplotlib/pyplot.py in post_execute()
    146 
    147             def post_execute():
--> 148                 if matplotlib.is_interactive():
    149                     draw_all()
    150 

AttributeError: 'NoneType' object has no attribute 'is_interactive'
```
Not sure why the error has to do with 'pdf'?
Thanks for taking a look

**Comment by vsoch** [comment]
I think this is one of the changes to the internal representation of the PixelData with the updated pydicom - definitely something we can fix! @darcymason before I dig into this does this ring any bells for what might be up?

**Comment by mobbSF** [comment]
fwiw, after commenting out the `.tostring()`, as in `dicom.PixelData = self.clean#.tostring()`, things continue until an exception for a bad tag in the dicom

**Comment by vsoch** [comment]
The background is that the representation of the pixel data had a big change between the older and current version of pydicom, and likely we have to obtain / set it differently than before. I don't know off the top of my head the correct way to go about it but can look into it later.

**Comment by vsoch** [comment]
hey @mobbSF ! We can address the matplotlib issue next, but for now give a look at this PR to see if the pixel data issue still arises: https://github.com/pydicom/deid/pull/53

Basically, I looked at the new docs to see how pixel data should be used:

https://pydicom.github.io/pydicom/stable/working_with_pixel_data.html

I'm not sure if this fixes it up, but thought it would be worth a shot!

**Comment by vsoch** [comment]
I'm going to close this issue here because the error in question was fixed by #58 (see #57 for new discussion around the issue) and there has been no more discussion here. Thanks!
